src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1359,13 +1359,4 @@
   <data name="UseAsSpanInsteadOfRangeIndexerOnAnArrayCodeFixTitle" xml:space="preserve">
     <value>Use `{0}` instead of Range-based indexers on an array</value>
   </data>
-  <data name="RuntimePlatformCheckTitle" xml:space="preserve">
```

Comment:
@buyaa-n Once your analyzer is ready, you can revert this commit and add correct values for these resource strings.